integrations/upgrade/js-config.test.ts

```diff
@@ -30,6 +30,7 @@ test(
                 400: '#f87171',
                 500: 'red',
               },
+              superRed: '#ff0000',
```

Comment:
Can we also add a test to ensure that a named modifier is actually converted to kebab-case as well?